src/Symfony/Component/Cache/CacheItem.php

```diff
@@ -100,6 +100,30 @@ public function expiresAfter($time)
     }
 
     /**
+     * Validates a cache key according to PSR-6.
+     *
+     * @param string $key The key to validate.
+     *
+     * @return string $key if it is valid.
+     *
+     * @throws InvalidArgumentException When $key is not valid.
+     */
+    public static function validateKey($key)
```

Comment:
> Never!

In theory. But since we return the value, we could modify it unawarely (10 minutes ago yo fixed a similar problem: https://github.com/symfony/symfony/pull/18603). This couldn't happen with a proper true/false validator.